xml/installation-installation-ironic-ironic_multi_control_plane.xml

```diff
@@ -23,16 +22,16 @@
   </para>
   <para>
    You need to set the environment variable
-   <emphasis role="bold">OS_REGION_NAME</emphasis> to the ironic region in
-   baremetal control plane. This will setup the ironic flat networking in
-   neutron.
+   <emphasis role="bold">OS_REGION_NAME</emphasis> to the Ironic region in
```

Comment:
If you are replacing such things -- please add an entity for "Ironic" and us it, also use the existing entity for "Neutron" (and those for "Swift", "Glance", etc. later on).